### ISSUE TITLE
Emoji-presentation ASCII digits are white with no border!

### ISSUE THREAD

**Issue by behdad** [googlei18n/noto-emoji#152]
See eg.:
https://github.com/googlei18n/noto-emoji/blob/master/png/128/emoji_u0030.png

I discovered this when I implemented them in Pango and tried, and saw empty space.  Turned out it was a fully-white zero shape on my white window background...

cc @roozbehp

**Comment by dougfelt** [comment]
This is the way they've always been.  They're there so they can combine
with combining enclosing keycap, since the sequence has an emoji
presentation.

It's up to Roozbeh I think.

**Comment by behdad** [comment]
Oh I thought we do that using GSUB ligatures.

**Comment by behdad** [comment]
Ok I see what you mean.  The glyphs are not really used to be composited on top of keycap glyph though.  They should really get a border.

**Comment by dougfelt** [comment]
Yeah, the glyphs are kind of pointless for the intended use. Anyway, if they need to be redrawn @roozbeh should file a bug so we can find someone to do it.

**Comment by roozbeh** [comment]
@roozbehp ^

**Comment by roozbehp** [comment]
The glyphs are white because when I created them, I took the ones from the keycaps and just took away the background that came from the keycap.

Since these would actually be triggered when someone uses the digits with the emoji variation selectors, I think it's best we show them in a color that's recognizable in more backgrounds, and also has a border. We can with either the color of regional indicators, or the color of copyright and trademark signs.
